### ISSUE TITLE
Anchor point error

### ISSUE THREAD

**Issue by Psychodeus** [ToroCraft/ToroHealth#168]
Values for hud.anchorPoint
TOP_LEFT, TOP_CENTER, TOP_RIGHT, BOTTOM_LEFT, BOTTOM_CENTER, BOTTOM_RIGHT

Only TOP_LEFT works.

**Comment by Sparkzzzzzz** [comment]
Yes same issue, plus how do I only get the bar above the mobs and not the dialog box thing?

**Comment by Sparkzzzzzz** [comment]
Config just isn't updating

**Comment by Psychodeus** [comment]
Type and enter this `%appdata%/.minecraft/config` at **Win + R**

Then find the **torohealth.json**
Then change the code to be like mine:

> {
  "watchForChanges": true,
  "hud": {
    "distance": 60,
    "x": 4.0,
    "y": 4.0,
    "scale": 1.0,
    "hideDelay": 20,
    "anchorPoint": "TOP_LEFT",
    "showEntity": true,
    "showBar": true,
    "showSkin": true,
    "onlyWhenHurt": false
  },
  "bar": {
    "damageNumberType": "LAST",
    "friendColor": "#00ff00",
    "friendColorSecondary": "#008000",
    "foeColor": "#ff0000",
    "foeColorSecondary": "#800000"
  },
  "inWorld": {
    "mode": "WHEN_HOLDING_WEAPON",
    "distance": 60.0,
    "onlyWhenLookingAt": true,
    "onlyWhenHurt": false
  },
  "particle": {
    "show": true,
    "damageColor": "#ff0000",
    "healColor": "#00ff00",
    "distance": 60
  }
}

But for the anchorPoint set value it to whatever alternative other than TOP_LEFT ('cause it's not working beside this anyway).
Restart game. Done.
